controller/iconcontroller.php

```diff
@@ -159,12 +159,19 @@ public function getLocalIconList() {
 				$icon['mimetype'] = mime_content_type($iconPath);
 				$icon['url'] = $this->urlGenerator->linkTo('passman', $path[1]);
 				$icon['pack'] = $pack;
+				$icon['data'] = base64_encode(file_get_contents($iconPath));
 				if (!isset($icons[$pack])) {
 					$icons[$pack] = [];
 				}
 				$icons[$pack][] = $icon;
 			}
 		}
-		return new JSONResponse($icons);
+
+		$offset = 3600 * 24 * 30;
```

Comment:
Change this for a precalculated constant, perhaps leaving the calculation on a comment for readability

Reply:
changed with b970780b4fda8288c682e0905bcab82cb938b222